tests/data/test_minute_bars.py

```diff
@@ -55,27 +57,24 @@
 TEST_CALENDAR_STOP = Timestamp('2015-12-31', tz='UTC')
 
 
-class BcolzMinuteBarTestCase(TestCase):
+class BcolzMinuteBarTestCase(WithTradingSchedule, WithInstanceTmpDir,
+                             ZiplineTestCase):
 
     @classmethod
-    def setUpClass(cls):
-        cls.env = TradingEnvironment()
-        all_market_opens = cls.env.open_and_closes.market_open
-        all_market_closes = cls.env.open_and_closes.market_close
-        indexer = all_market_opens.index.slice_indexer(
-            start=TEST_CALENDAR_START,
-            end=TEST_CALENDAR_STOP
+    def init_class_fixtures(cls):
+        super(BcolzMinuteBarTestCase, cls).init_class_fixtures()
+        trading_days = get_calendar('NYSE').trading_days(
```

Comment:
Could `trading_days` be provided by default by the `WithTradingSchedule` fixture?